terminologies/technique/lightSheetMicroscopy.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+  "@id": "https://openminds.ebrains.eu/controlledTerms/technique/lightSheetMicroscopy",

```

Comment:
it appears to me that the term "light-sheet fluorescence microscopy" is more used. And I've learned that this is also referred to as "single plane illumination microscopy", or "orthogonal plane fluorescence optical sectioning microscopy". Each are actually different techniques with different resolutions. At least that's what I've found